openapi/platform/platform.yaml

```diff
@@ -3935,7 +3939,19 @@ components:
               dateValue:
                 type: string
                 description: Required when the field type is `date`
-                format: date-time
+                format: date
+              dropdownValue:
+                type: string
+                description: Required when type is `dropdown`. The possible values can be discovered using the administration panel.
+              currencyValue:
+                type: integer
+                description: |-
+                  Required when the field type is `currency`.
+
+                  This field represents the cents (or the smallest part of the currency).
```

Comment:
How does the user know what currency it is in? Since we have the description and title from the field schema should we have the currency as well?

Reply:
(Answering this question made me realize I left this field it out of the model, fixing it) 

They can see on administration panel and the `/userCustomFields` and `/salesAccountsCustomFields` returns it.

```
                {
                    "fieldId": "FieldID-da94e32c-1e3e-4fbe-a99d-3eb2ec4f1c11",
                    "title": "Test Currency",
                    "description": "Desc Currency",
                    "currencyValue": 5000,
                    "fieldType": "currency",
                    "currencyCode": "CAD"
                }
```